transform-plugins/src/main/java/co/cask/hydrator/plugin/JSONParser.java

```diff
@@ -86,12 +169,17 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
     @Description("Input Field")
     private String field;
 
+    @Name("mapping")
+    @Description("Mapping complex JSON to fields using JSON Path expressions")
+    private String mapping;
```

Comment:
mark this @Nullable, to indicate that it is optional field? I think that is propagated to the UI then.
@gsps1, can you confirm? I remember you talking about that earlier this week.